go/vt/vttablet/tabletmanager/rpc_backup.go

```diff
@@ -49,22 +44,28 @@ func (agent *ActionAgent) Backup(ctx context.Context, concurrency int, logger lo
 	if !allowMaster && currentTablet.Type == topodatapb.TabletType_MASTER {
 		return fmt.Errorf("type MASTER cannot take backup. if you really need to do this, rerun the backup command with -allow_master")
 	}
-
-	tablet, err := agent.TopoServer.GetTablet(ctx, agent.TabletAlias)
```

Comment:
What's the reason for removing this?

Reply:
I think it was intentionally fetching the latest tablet record to check if it's master, in case it's different than the local one?

Reply:
I guess we may have to fetch a second time after acquiring the action lock, since it may have changed by then.

Reply:
good catch, that was unintentionally removed while moving the code around.